packages/icons/package.json

```diff
@@ -39,6 +39,7 @@
     "provenance": true
   },
   "files": [
-    "css"
+    "css",
+    "saltIcons.css"
```

Comment:
Is it better to put this inside existing `css` folder?

And i realized that we may need some additional comment on top of `salt-icon.css` and this new `saltIcons.css` to make it clearly to the user, one is for no injection usage